### DISCUSSION TITLE
How to configure the default tombstone offset of node?

### DISCUSSION THREAD

**Discussion by Ljzn** [mobilecoinfoundation/mobilecoin#3950] [Q&A] [answered]
Hi, mobilecoin team. The default tombstone offset is too small for our use case, is that configurable? We want keep letting node to auto sets the tombstone, with larger offset. Thanks!

**Comment by nick-mobilecoin** [comment, marked as answer]
The tombstone block for the block chain itself is, https://github.com/mobilecoinfoundation/mobilecoin/blob/21d3af4d7122bb4bfeae7ff4cff3d78b9b1a323b/transaction/types/src/constants.rs#L35
It can be specified per transaction via https://github.com/mobilecoinfoundation/mobilecoin/blob/21d3af4d7122bb4bfeae7ff4cff3d78b9b1a323b/transaction/builder/src/transaction_builder.rs#L723

If however this is a fog transaction it's most likely limited by https://github.com/mobilecoinfoundation/mobilecoin/blob/21d3af4d7122bb4bfeae7ff4cff3d78b9b1a323b/fog/ingest/server/src/config.rs#L82 which defaults to 10 blocks

**Reply by finewhitechic** [reply]
How do I make a token usd transaction
